course-content/02_core_data_structure_concepts.qmd

```diff
@@ -230,6 +234,52 @@ What would these expressions return?
 7. If a part of the slice is out of range, the operation does not fail. `full_name[:]` gives the same result as `full_name[0:42]`, and the same result as `full_name[:103]`.
 :::
 
+### Dictionaries
+
+Another object that exists in Python is the `dictionary`. It is similar to a list in that it can hold a variety of different types of objects inside of it. However an important difference is in how we access these objects. With a list (or string), we have an ordered arrangement of items that we access with an **integer index**. However, we access the **values** in a dictionary with a **key**, which can be anything we want.
+
+Let's build a dictionary, which is denoted in Python with curly `{}` brackets:
+
+```{python}
+my_dict = {
+    'first_key': 'some value',
+    'A': ['a', 'differerent', 'type', 'of', 'object'],
+    2: False
```

Comment:
Interesting that the keys don't have to be same type (i.e., a string). Can you / do you ever look up more than one element of a dict?

Reply:
There's no specifically built in way to do it (you can't do my_dict['first_key', 'A']  or anything). It is doable, but more complicated than worth mentioning here I'd say.